caddy/caddymain/run.go

```diff
@@ -105,11 +105,12 @@ func Run() {
 	}
 
 	if validate {
-
-		_, err := caddy.ValidateCaddyFile(caddyfileinput, nil)
+		justValidate := true
+		_, err := caddy.ValidateAndExecuteDirectives(caddyfileinput, nil, justValidate)
 		if err != nil {
 			mustLogFatalf(err.Error())
 		}
+		log.Println("[INFO] Caddyfile Valid")
```

Comment:
I know you asked to remove the INFO log earlier.  But I felt that if the user starts with -log they should receive an INFO log item